scripts/importing-Exporting-Config.js

```diff
@@ -0,0 +1,480 @@
+    {
+        var season = seasonList[i]; //if season was included, it probably had data; look for element/pit/event/team data for those seasons
+
+        var savedSeasons = getCookie('seasonList');
+        if (!savedSeasons.includes(season)) //if season is new, add it to the seasonList cookie
+        {
+            savedSeasons += season + SEASON_LIST_SEPARATOR;
+            setCookie('seasonList',savedSeasons);
+        }
+
+        //the seaon object in JSON file
+        var seasonObjValues = obj[season]; //has keys like 'elements' and 'pit questions'...
+    
+        //set element/value cookies
+        var elementsObj = seasonObjValues[EXPORT_CONFIG_ELEMENTS_KEY];
+        var elementKeys = Object.keys(elementsObj);
+        for (var j = 0; j < elementKeys.length; j++)
+        {
+            var elementName = elementKeys[j]; //only has one kv pair
+            var elementValue = elementsObj[elementName];
+
+            var cookieName = season_config_cookie_name(season,elementName);
+            setCookie(cookieName,elementValue,750);
+        }
+
+        //set pitQuestion cookie
+        var pitObjArray = seasonObjValues[EXPORT_CONFIG_PIT_QUESTIONS_KEY];
+        var pitCookieName = `${season} pitQuestions`
+        var pitCookieValString = '';
+        for (var j = 0; j < pitObjArray.length; j++)
+        {
+            pitCookieValString += pitObjArray[j] + COOKIE_QUESTION_SEPARATOR;
+        }
+        setCookie(pitCookieName,pitCookieValString,750);
+
+        //set event cookie
+        var eventObjArray = seasonObjValues[EXPORT_CONFIG_EVENTS_KEY];
+        var eventCookieName = `${season} events`;
+        var eventCookieValString = '';
+        for (var j = 0; j < eventObjArray.length; j++)
+        {
+            eventCookieValString += eventObjArray[j] + EVENT_LIST_COOKIE_SEPARATOR;
+        }
+        setCookie(eventCookieName,eventCookieValString,750);
+
+        //set team data local storage objects
+        processTeamData(season,seasonObjValues,EXPORT_CONFIG_TEAM_DATA_KEY);
+
+    }
+    loadSeasonRadioList() // 'confirmation' to the user that the desired seasons were added
+    $('#importConfirmation').text('Data imported.');
+}
+
+function processTeamData(season,originalObject,team_Data_Obj_Key)
+{
+    var team_Data_Obj_Entry = originalObject[team_Data_Obj_Key];
+    var events_In_Team = Object.keys(team_Data_Obj_Entry);
+
+    //console.log(events_In_Team);
+    for (var i = 0; i < events_In_Team.length; i++)
+    {
+        var curr_Event = events_In_Team[i];
+        var curr_Event_Entries = team_Data_Obj_Entry[curr_Event];
+        var teams_In_Curr_Event = Object.keys(curr_Event_Entries); //in format 6321:rouse
+        var cookie_Team_Obj = {};
+
+        for (var j = 0; j < teams_In_Curr_Event.length; j++)
+        {
+            var curr_Team_Key = teams_In_Curr_Event[j];
+            var curr_Team_Num = curr_Team_Key.split(':')[0];
+            var curr_Team_Name = curr_Team_Key.split(':')[1];
```

Comment:
what if the team name has a `:` in it?

might be better to key off of team numbers and have `"name": "..."` be one of the values you store (in addition to the match data and pit data).